jni/ffigen.yaml

```diff
@@ -28,6 +28,7 @@ functions:
     - 'GetJniContext'
     - 'setJniGetters'
     - 'jni_log'
+    - 'Java_com_github_dart_1lang_jni_PortContinuation__1resumeWith'
```

Comment:
Does the above comment also hold for this?

Nit: above comment does not have punctuation. (Ditto for other comments in the file.)